src/test/ui/suggestions/suggest-assoc-fn-call-with-turbofish.fixed

```diff
@@ -0,0 +1,29 @@
+// run-rustfix
+
+struct GenericAssocMethod<T>(T);
+
+impl<T> GenericAssocMethod<T> {
+    fn default_hello() {}
+    fn self_ty_hello(_: Self) {}
+    fn self_ty_ref_hello(_: &Self) {}
+}
+
+fn main() {
+    // Test for inferred types
+    let x = GenericAssocMethod(33);
+    // This particular case is unfixable without more information by the user,
```

Comment:
Can you be explicit about needing more information? If this isn't a `MachineApplicable` diagnostic, then it shouldn't be marked `MachineApplicable` I think.

Reply:
The case of `x.default_hello()` (or any case where the type can't be inferred) emits a `HasPlaceholders` diagnostic and suggests `GenericAssocMethod::<_>::default_hello()`